docs/ReleaseNotes.md

```diff
@@ -47,8 +48,8 @@ Constructors of the `RecordQueryUnionPlan` and `RecordQueryIntersectionPlan` hav
 
 ### 2.7.76.0
 
-* All changes from version [2.7.73.20](#277320)
 * **Bug fix** Revert fix for #646 (in [2.7.75.0](#27750)) until a complete resolution is worked out [(PR #654)](https://github.com/FoundationDB/fdb-record-layer/pull/654)
+* All changes from version [2.7.73.20](#277320)
```

Comment:
This change is actually unrelated--it is just fixing the release notes here to be in the same order as the others (i.e., all fixes, then the "all changes from version" message). In theory, it could be a separate PR.